lib/internal/streams/operators.js

```diff
@@ -263,7 +263,7 @@ function toIntegerOrInfinity(number) {
   return number;
 }
 
-function drop(number, options) {
+function drop(number, options = {}) {
```

Comment:
We don't need to create a throw away object here.
```suggestion
function drop(number, options = undefined) {
```